Blaise.Tests.Helpers/Browser/BrowserHelper.cs

```diff
@@ -79,5 +90,100 @@ private static ChromeDriver CreateChromeDriver()
 
             return new ChromeDriver(BrowserConfigurationHelper.ChromeDriver, chromeOptions);
         }
+
+        //AP changes - 20-03-2023
+        public static IWebElement WaitForElement(By by, bool checkIfClickable = false, int maxAttempts = 50)
+        {
+            IWebElement element = null;
+            var attempts = 0;
+
+            while (element == null && attempts < maxAttempts)
```

Comment:
If max attempts is reached then it would be good to get an error to indicated that this is the case? Perhaps it should raise a custom exception?

Reply:
it now logs a NoSuchElementException